### ISSUE TITLE
Extend the tests (tox, Travis CI) to Django 1.10

### ISSUE THREAD

**Issue by bittner** [jcassee/django-analytical#104]
Extending the tests to Django 1.10 is more than simply updating [tox.ini](https://github.com/jcassee/django-analytical/blob/master/tox.ini#L5) and [.travis.yml](https://github.com/jcassee/django-analytical/blob/master/.travis.yml#L23): A DjangoTemplates backend needs to be configured. A tox run yields errors for Django 1.10:

```
django.core.exceptions.ImproperlyConfigured: No DjangoTemplates backend is configured.
```

Related Django docs
---------------

- [Topics: Templates](https://docs.djangoproject.com/en/1.10/topics/templates/#configuration)
- [Reference: Templates](https://docs.djangoproject.com/en/1.10/ref/templates/upgrading/#the-templates-settings)  (upgrading)

**Comment by pjdelport** [comment]
#114 should fix this.

**Comment by jcassee** [comment]
Fixed by #144
